src/huntsman/pocs/camera/pyro/client.py

```diff
@@ -104,6 +112,14 @@ def cooling_power(self):
     def is_exposing(self):
         return self._proxy.get("is_exposing")
 
+    @is_exposing.setter
+    def is_exposing(self, is_exposing):
+        """Set or clear the remote exposure event."""
+        if is_exposing:
+            self._exposure_event.set()
+        else:
+            self._exposure_event.clear()
+
```

Comment:
I suspect this is rendered (almost) unnecessary by my changes in https://github.com/panoptes/POCS/pull/1026

Looks like a workaround for the `Camera` base class (and hence the Pyro client) trying to assign to `is_exposing` when as far as the Pyro service (and SDK cameras) are concerned `is_exposing` is read only.

Reply:
I think it was necessary to get the tests passing, but let's worry about this when we revisit the camera class.

Reply:
Yes, it would have been necessary, but what I'm suggesting is that now those other changes have been merged into POCS it's probably no longer necessary (and will probably actually break stuff).